scripts/vktrace_file_generator.py

```diff
@@ -3330,7 +3330,7 @@ def GenerateTraceVkPacketsHeader(self):
                             or (p.name in ['pFeatures'] and 'nvx' in p.type.lower())
                             or (p.name in ['pFeatures', 'pProperties','pFormatProperties','pImageFormatInfo','pImageFormatProperties','pQueueFamilyProperties',
                                            'pMemoryProperties','pFormatInfo','pSurfaceFormats','pMemoryRequirements','pInfo',
-                                           'pSparseMemoryRequirements','pSurfaceCapabilities'] and '2khr' in p.type.lower())
+                                           'pSparseMemoryRequirements','pSurfaceCapabilities'] and '2' in p.type.lower())
                             or (p.name in ['pSurfaceCapabilities'] and '2ext' in p.type.lower())):
```

Comment:
This line is now obsolete too? Sorry, I should have noticed this when I reviewed this earlier.

Reply:
yes, seems like it. Sorry was focus in fixing the specific issue which causing regression playback on a few game titles, didn't notice it actually covers most of the cases :) please refer to this commit [05b5558](https://github.com/LunarG/VulkanTools/pull/814/commits/05b55587a38c8cff20231d7ae853e5a49a42f44f)